native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -99,12 +76,14 @@ extension ViewController: ApplicationContractView {
     func setStations(stations: [Station]) {
         stationData = stations;
             
-        if (!stations.isEmpty) {
-            presenter.setDepartureStation(station: stations[0])
-            presenter.setArrivalStation(station: stations[0])
+        departureStation.optionArray = stationData.map{$0.name}
```

Comment:
Maybe use $0.toString rather than .name to unify with Android?

Reply:
done using $0.description()